src/test/java/com/salesforce/dynamodbv2/mt/mappers/MtAmazonDynamoDbCompositeTest.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root  or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+package com.salesforce.dynamodbv2.mt.mappers;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+import static org.mockito.Mockito.mock;
+
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB;
+import com.google.common.collect.ImmutableList;
+import com.salesforce.dynamodbv2.mt.context.MtAmazonDynamoDbContextProvider;
+import io.micrometer.core.instrument.MeterRegistry;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.stream.Stream;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+class MtAmazonDynamoDbCompositeTest {
```

Comment:
I think a happy path test case demonstrating the toggle `fromContextToDelegate` appropriately sends data to the different delegates, maybe validating on a multi tenant scan across the two shared tables would be nice